test/unit/openscap_host_test.rb

```diff
@@ -4,19 +4,9 @@ class OpenscapHostTest < ActiveSupport::TestCase
   setup do
     disable_orchestration
     User.current = users :admin
-    Setting[:token_duration] = 0
     Scaptimony::Policy.any_instance.stubs(:ensure_needed_puppetclasses).returns(true)
   end
-
-  test 'Host has policy' do
```

Comment:
Rebase on https://github.com/theforeman/foreman_openscap/pull/108 and you don't have to throw it away (at least works for me)